src/CTA.Rules.PortCore/ProjectSpecificPort/WCF/Constants.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections.Generic;
 
-namespace CTA.Rules.Update.WCF
+namespace CTA.Rules.PortCore.WCF
```

Comment:
Why is this modified while the file is still in the Update project?

Reply:
This file was moved to the PortCore project. Github's diff view tries to show that, but it gets cut off depending on browser window size